gitautosync/pull_from_remote.py

```diff
@@ -0,0 +1,76 @@
+import os
+import subprocess
+
+
+def pull_from_remote(repo_name, branch_name, sync_path, account, domain):
```

Comment:
Is there a reason we're getting repo name, account and domain separately instead of just getting git url?